config/locales/de.yml

```diff
@@ -47,56 +47,56 @@ de:
         precision: 0
         format: '%u%n'
   homepage:
-    intro: "<strong>codebar</strong> is a charity that facilitates the growth of a diverse tech community by running free regular programming workshops for minority groups in tech."
+    intro: "<strong>codebar</strong> ist eine Wohltätigkeitsorganisation, die das Wachstum einer vielfältigen Tech-Community fördert, indem sie regelmäßig kostenlose Programmier-Workshops für Minderheiten in der Tech-Branche anbietet."
     explanation: "Unser Ziel ist unterrepräsentierten Menschen zu programmieren in einer sicheren und kollaborative Umgebung lernen und erweitern ihre Karrierechancen zu ermöglichen. Um dies zu erreichen betreiben wir kostenlose regelmäßige Workshops, regelmäßige Einzelveranstaltungen und versuchen, die Möglichkeiten für unsere Schüler die Technologie und Programmierung zugänglicher zu schaffen."
```

Comment:
My take on this one. I'd also suggest `Teilnehmer:innen` for `Students`

`Unser Ziel ist es Menschen, die in der IT-Branche unterrepräsentiert sind, technologische Fähigkeiten in einer sicheren und kollaborative Umgebung zu erlernen und dadurch ihre Karrierechancen zu erhöhen. Dafür veranstalten wir regelmäßig kostenlose Workshops und Einzelveranstaltungen, in denen die Möglichkeiten von Technologie und Programmieren für unsere Teilnehmer:innen zugänglich gemacht werden.`